.gitignore

```diff
@@ -10,6 +10,9 @@ staging/kos/local-binaries
 staging/kos/deploy/main/50-rs-*.yaml
 staging/kos/deploy/main/50-ds-*.yaml
 
+# The generated tls certificates and keys, and the dir holding them
+staging/kos/etcd-tls/certs
```

Comment:
It would be better if the certs were not kept in the filesystem branch managed by git.  I suggest something like `~/.kos/etcd-tls`.